app/uk/gov/hmrc/gform/sharedmodel/formtemplate/Section.scala

```diff
@@ -156,12 +156,16 @@ object AcknowledgementSection {
 
 }
 
-case class AcknowledgementSectionPdf(header: Option[SmartString], footer: Option[SmartString])
+case class SectionPdf(
```

Comment:
Why was this renamed? `SectionPdf` feels like a strange name, it is hard to guess what does it mean.

Reply:
We should reuse it both case classes `AcknowledgementSection` and `SummarySection`. I'll try to find a better name for it.